### DISCUSSION TITLE
Pullup resistor?

### DISCUSSION THREAD

**Discussion by JGeeforce** [huizebruin/s0tool#17] [General] [answered]
Hi,

I see your website in renewed and I miss the part where you had the pictures about the pull up resistor. Over which wires do I have to place the pull up resistor?

Thanks!

**Comment by huizebruin** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/62996429/160697202-76e89c65-d495-4cd7-8c36-0c98bc51cedd.png)

Did you mean this one?
[from my website Huizebruin.nl](https://www.huizebruin.nl/home-assistant/esphome/watermeter-uitlezen-in-home-assistant-met-esphome)
